setup.cfg

```diff
@@ -20,7 +20,7 @@ classifiers =
 packages = find:
 install_requires =
     PyCifRW~=4.4
-    aiida-core>=2.1,<3
+    aiida-core[atomic_tools]>=2.1,<3
```

Comment:
Please let's not do this. This is a big dependency, and pymatgen was giving us troubles before. If this is because of ase, please add ASE separately.